xml/System.Collections.Generic/SortedDictionary`2+ValueCollection.xml

```diff
@@ -537,13 +537,13 @@
   
  If <xref:System.Collections.IEnumerator.MoveNext%2A> passes the end of the collection, the enumerator is positioned after the last element in the collection and <xref:System.Collections.IEnumerator.MoveNext%2A> returns `false`. When the enumerator is at this position, subsequent calls to <xref:System.Collections.IEnumerator.MoveNext%2A> also return `false`. If the last call to <xref:System.Collections.IEnumerator.MoveNext%2A> returned `false`, <xref:System.Collections.Generic.IEnumerator%601.Current%2A> is undefined. You cannot set <xref:System.Collections.Generic.IEnumerator%601.Current%2A> to the first element of the collection again; you must create a new enumerator instance instead.  
   
- An enumerator remains valid as long as the collection remains unchanged. If changes are made to the collection, such as adding, modifying, or deleting elements, the enumerator is irrecoverably invalidated and its behavior is undefined.  
+ An enumerator remains valid as long as the collection remains unchanged. If changes are made to the collection, such as adding, modifying, or deleting elements, the enumerator is irrecoverably invalidated and the next call to <xref:System.Collections.IEnumerator.MoveNext%2A> or <xref:System.Collections.IEnumerator.Reset%2A> throws an <xref:System.InvalidOperationException>. If the collection is modified between calling <xref:System.Collections.IEnumerator.MoveNext%2A> method and obtaining the value of the <xref:System.Collections.Generic.IEnumerator%601.Current%2A> property, <xref:System.Collections.Generic.IEnumerator%601.Current%2A> returns the element that it is set to, even if the enumerator is already invalidated.  
   
  The enumerator does not have exclusive access to the collection; therefore, enumerating through a collection is intrinsically not a thread-safe procedure. To guarantee thread safety during enumeration, you can lock the collection during the entire enumeration.  To allow the collection to be accessed by multiple threads for reading and writing, you must implement your own synchronization.  
   
  Default implementations of collections in the <xref:System.Collections.Generic> namespace are not synchronized.  
   
- This method is an O(1) operation.  
+ This method is an O(log n) operation where n is a number of elements in a collection.
```

Comment:
I'm pretty sure this change (in all locations on a `MoveNext` method) is incorrect. `MoveNext` is O(1). ping @ianhays @safern for verification.

Reply:
It should be for `GetEnumerator` method. Did I  miss it?

Reply:
@BillWagner could you please have a look at this one more time. It looks that mere misunderstanding is making this PR stale: the [MoveNext method](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.sorteddictionary-2.valuecollection.enumerator.movenext?view=netcore-2.0) doesn't have any remarks on its time complexity, the changes in questions are done for the `GetEnumerator` methods. I've fixed the time complexity based on the fact that `SortedDictionary` is implemented based on `SortedSet`, which has O(log n) [GetEnumerator](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.sortedset-1.getenumerator?view=netcore-2.0) method.

Reply:
Sorry about letting this get stale. I've been reviewing this morning.